### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by jasonacox** [jasonacox/Powerwall-Dashboard#390] [Announcements]
I'm grateful for this incredible community! Thanks to all of you who contribute, open issues, share experiences, help answer questions and make suggestions. You are what makes this project so fun!

The best is yet to come... 

Cheers!

**Comment by hulkster** [comment]
Thank YOU (and others) who have created a really nifty product!

**Comment by mcbirse** [comment]
Thank you Jason, and the many others involved, this is such a great project! I have enjoyed every minute of the (now countless) hours spent adding my contribution where I can, and have learnt so much along the way thanks to this fantastic community!

The result of so many willing to share is truly something amazing, and the project just keeps getting better and better! 👍
